.github/workflows/ci.yml

```
@@ -33,3 +35,6 @@ jobs:
 
       - name: Run Tests
         run: npm test
+
+      - name: Build Application # S'assurer que le build fonctionne
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)